web/src/emoji_picker.js

```diff
@@ -189,21 +188,22 @@ function refill_section_head_offsets($popover) {
 }
 
 export function reactions_popped() {
-    return $current_message_emoji_popover_elem !== undefined;
+    return emoji_popover_instance;
 }
```

Comment:
So am I correct that this is actually not just for the reactions special case? 

First, this function should return a bool, not the instance -- that seems unintentionally changed here.

And then let's add a new last commit to rename this function to `emoji_picker.is_open`, since that's a much clearer description of what it checks.

Reply:
> So am I correct that this is actually not just for the reactions special case?

Reactions is just what we call the emoji popover; it's not a special case.

> First, this function should return a bool, not the instance -- that seems unintentionally changed here.

Is it okay to return something like `Boolean(emoji_popover_instance)`? I'm not sure that a comparison with `undefined` might be 100% safe (if someone sets it to `null` somewhere, for example).

> And then let's add a new last commit to rename this function to emoji_picker.is_open, since that's a much clearer description of what it checks.

Sure!

Reply:
> Is it okay to return something like Boolean(emoji_popover_instance)? I'm not sure that a comparison with undefined might be 100% safe (if someone sets it to null somewhere, for example).

I think `=== undefined` is probably better, or `=== null` if that's the convention being used in this file, which might be better. But I don't see why we'd be worrying about writing the code to expect both possibilities when we're very much writing all the logic that sets this variable right here in this file.

Reply:
I left this as `Boolean()` but should flag you put the rename of the function itself in the main Bootstrap commit -- which meant that commit didn't run.

Reply:
Hmm, I thought about making this renaming a separate commit. I might have missed something while rebasing.